Added Minoo-M-Advice

minoomayfield opened the pull request.

redrambles commented:
Hey @minoomayfield!

This is great - the only thing we are missing is a link to your advice page from the main index.html file, so that folks can find your words of wisdom, once your code is merged in. Students will typically add their link to the top or the bottom of that long list. :)

Once you add your link, add and commit your changes and then push your branch up once more, which will automatically update your pull request. Then we should be good to merge in your code!

Let us know if you have any questions!

Cheers,
Ann

minoomayfield commented:
Thank you! I added my link to the index file, added & committed changes & did git push origin advice. i see now that this branch has conflicts that must be resolved. is there anything else i need to do on my end? thank you!!

redrambles commented:
Hey @minoomayfield !

Great!! Thanks for making that change - your code is merged in! 👍

Cheers,
Ann :)
